agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/wasbootstrap/configuration/Configuration.java

```diff
@@ -418,4 +420,15 @@ public void validate() throws FriendlyException {
     // transient so that Moshi will ignore when binding from json
     public transient long lastModifiedTime;
 
+    public static class ProfilerConfiguration {
+        public int configPollPeriod = -1;
+        public int periodicRecordingDuration = -1;
+        public int periodicRecordingInterval = -1;
```

Comment:
we've been putting the default values here, which has been convenient

also, I really like to put time unit at the end of config names so that there's no confusion

```suggestion
        public int configPollingIntervalSeconds = 60;
        public int periodicRecordingDurationSeconds = 120;
        public int periodicRecordingIntervalSeconds = 60 * 60;
```